sycl/test/lit.cfg

```diff
@@ -5,6 +5,7 @@ import platform
 import re
 import subprocess
 import tempfile
+from distutils.spawn import find_executable
```

Comment:
I'm far from being a Python/LIT infra expert, so I'd appreciate a suggestion if someone knows a better way to do this